host-bmc/host_pdr_handler.cpp

```diff
@@ -935,7 +1026,7 @@ void HostPDRHandler::_setHostSensorState()
                 sensorRearm.byte = 0;
                 uint8_t tid = std::get<0>(terminusInfo);
 
-                auto instanceId = instanceIdDb.next(mctp_eid);
+                auto instanceId = instanceIdDb.next(mctpEid);
```

Comment:
please free the same mctpEid in line 1039

Reply:
done